views/profile.handlebars

```diff
@@ -83,11 +80,10 @@
             </div>
           </div>
 
-
           <!-- Submit -->
           <a id="submitProfile" class="waves-effect waves-light btn">Submit</a>
         </form>
-      </div>
+
      </div>
    </div>
  </div>
```

Comment:
Add in handlebars notation for member names and emails, based on who logged in. This might need to be coordinated with Jean and authentication. 

For now I think this is good.

Reply:
Removing handlebars from this page as it is unnecessary